policy/datasets.yaml

```diff
@@ -1103,6 +1103,19 @@ makeBrighterFatterKernel_metadata:
     python: lsst.daf.base.PropertySet
     tables: raw
     template: brighterFatter_metadata/brighterFatter_metadata.yaml
+processStar_config:
+    description: "Configuration for the processStar task."
+    persistable: Config
+    python: lsst.atmospec.ProcessStarTaskConfig
+    storage: ConfigStorage
+    tables:
+    - raw
+    template: config/processStarConfig.py
+processStar_metadata:
+    persistable: PropertySet
+    storage: YamlStorage
+    python: lsst.daf.base.PropertySet
+    template: processStar_metadata/%(visit)08d_metadata.yaml
```

Comment:
On 1118, should that template have the 08 removed, or is that there to restrict it to 8 characters?